server/controllers/payroll/weekendConfig/index.js

```diff
@@ -49,24 +49,42 @@ function detail(req, res, next) {
 }
 
 // POST /WEEKEND_CONFIG
-function create(req, res, next) {
+async function create(req, res) {
```

Comment:
This is better!  We still need to handle errors though.  To make sure these errors are handled, please add back in "next()' and wrap all your code in a try/catch block.  Like this:
```js
async function create(req, res, next) {
  try {
    // all your code
  } catch (e) {
    next(e);
  }
}
```